### PR TITLE
test: selected facets module test

### PR DESCRIPTION
**Pull request by deepunbxd005**

### CODE REVIEW

File: src/modules/combinedFacets/__tests__/CombinedFacets.test.js [hunk 1]
```diff
@@ -142,3 +144,61 @@ test('Test Combined Facet category click', async () => {
...
+        </>
+    );
+
+    await waitFor(async () => {
+        expect(getByText('Scarpa - 18')).toBeInTheDocument();
+        fireEvent.click(getByText("Scarpa - 18"));
+    });
+    await waitFor(() => {
+        expect(container.getElementsByClassName('UNX-selectedFacets__container').length).toBe(1)
+        fireEvent.click(getByText("Clear"));
```
Comment (vikramgupta1988) [inline src/modules/combinedFacets/__tests__/CombinedFacets.test.js:202]: Also check after click on clear btn, UNX-selectedFacets__container length must be 0.

**Review by vikramgupta1988 — CHANGES_REQUESTED** [review]

**Review by vikramgupta1988 — APPROVED** [review]

File: src/modules/combinedFacets/__tests__/CombinedFacets.test.js (hunk 1 above)
Comment (deepunbxd005) [reply src/modules/combinedFacets/__tests__/CombinedFacets.test.js:202]: done

**Pull request merged**